client/src/app/users/user-filter/user-filter.component.html

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Why is this component a partial portion of a card and not the whole card?

Reply:
I suppose it could be different. The only reason I did it that way was because I was not aiming to change the look of the previous page, but I was wanting to pull the filtering out of the list.